crypto/sig/src/lib.rs

```diff
@@ -0,0 +1,125 @@
+#![no_std]
+
+//! Simplified API for using Schnorrkel in a deterministic manner, with simple
+//! ristretto public and ristretto private key-pairs.
```

Comment:
For audit purposes, it may be worth mentioning here that RistrettoPrivate is a wrapper of a Ristretto Scalar, and RistrettoPublic is a Ristretto Point